Introduce a kludge to make this work on android / windows

gillpeacegood opened the pull request:
Just thought this might be of interest to people if working on android/react-native-windows

gillpeacegood committed:
Introduce a kludge to make this work on android / windows

sghiassy commented:
Thanks @gillpeacegood for the PR. Sorry for the belated review. 

The PR looks promising. I certainly would like to improve Android and Windows performance, but the PR in it's current state is failing on CI because of linting issues. Can you run check the Travis CI output and fix any lint errors. Thx

sghiassy commented on lib/SGListView.js:
```diff
@@ -41,6 +41,31 @@ const PrivateMethods = {
     }
   },
 
+  makeFrame(cellData, childFrames, sectionID, rowID, horizontal, index) {
```
Can you add a description of this function using the javadoc format? What are we considering a `frame` here? Frames makes me thing we're generating a wrapper `<View />` , but it looks like we're returning `{ x, y, width, height, index }`. Would `calculateDimensionsForCell` be more descriptive?

ie.
```
/*
 * This function does...
 */

gillpeacegood committed:
Merge branch 'master' of https://github.com/sghiassy/react-native-sglistview

gillpeacegood committed:
Sort out lint and code review issues

sghiassy committed:
Delete SGListView.map

Trying to resolve conflicts in https://github.com/sghiassy/react-native-sglistview/pull/45

sghiassy committed:
Delete SGListView

Trying to resolve conflicts in https://github.com/sghiassy/react-native-sglistview/pull/45 without bugging you again ;)

sghiassy commented:
Thanks for updating the PR - merging
